File: 1º Semestre/Programação de Computadores/Lista de Exercicios 3/Python/10.py
"""
10. Elabore um algoritmo que solicite ao usuário uma palavra e um número inteiro que
indicará a quantidade de vezes que a palavra digitada será impressa na tela, um em cada
linha.
"""

strInput = str(input("Digite uma palavra: "))
intInput = int(input("Digite o repetições para a {}: ".format(strInput)))

for i in range(0, intInput):
    print(strInput)

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 3/C-C++/10.cpp
#include <iostream>
using namespace std;

/*
10. Elabore um algoritmo que solicite ao usuário uma palavra e um número inteiro que
indicará a quantidade de vezes que a palavra digitada será impressa na tela, um em cada
linha.
*/

int main(){
	int n;
	string p;
	
	cout << "Digite uma palavra: ";
	cin >> p;
	
	cout << "Digite um numero inteiro: ";
	cin >> n;
	
	for(int i = 1; i <= n; i++){
		cout << p << "\n";
	}	
}
